src/mplfinance/plotting.py

```diff
@@ -1070,6 +1075,8 @@ def _addplot_columns(panid,panels,ydata,apdict,xdates,config):
 def _addplot_apply_supplements(ax,apdict,xdates):
     if (apdict['ylabel'] is not None):
         ax.set_ylabel(apdict['ylabel'])
+    if (apdict['xlabel'] is not None):
```

Comment:
This will not work until the new kwarg is ***also*** added to `_valid_addplot_kwargs()` (as noted above).
Thanks.